_posts/2022-09-08-akka-license-change.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Regarding Akka's licensing change"
+date: 2022-09-08T08:00:00.000Z
+categories: news
+authors:
+- Chesnay:
+  name: "Chesnay Schepler"
+
+---
+
+On September 7th Lightbend announced a [license change](https://www.lightbend.com/blog/why-we-are-changing-the-license-for-akka) for the Akka project, the TL;DR being that you will need a commercial license to use future versions of Akka (2.7+) in production if you exceed a certain revenue threshold.
+
+Within a few hours of the announcement several people reached out to the Flink project, worrying about the impact this has on Flink, as we use Akka internally.
+
+The purpose of the blogpost is to clarify our position on the matter.
+
+Please be aware that this topic is still quite fresh, and things are subject to change.  
+Should anything significant change we will amend this blogpost and inform you via the usual channels.
+
+# Give me the short version
+
+Flink is not in any immediate danger and we will ensure that users are not affected by this change.
+
+The licensing of Flink will not change.
+
+We will not use Akka versions with the new license.
+
+# What's the plan going forward?
+
+_For now_, we'll stay on Akka 2.6, the current latest version that is still available under the original license.
```

Comment:
`For now` is intentionally ambiguous. Anything else (like a particular Flink version release) would be a pure guess.